app/templates/eventStudentView.html

```diff
@@ -0,0 +1,97 @@
+    {% endif %}
+    </div>
+    {% if isPastEvent %}
+        <p><font size="+2"><b>{{eventData.name}}</b> took place on <b>{{eventData.startDate}}</b> from <b>{{eventData.timeStart}}</b> to <b>{{eventData.timeEnd}}</b> and was located in <b>{{eventData.location}}</b>.</font></p>
+    {% else %}
+        <p><font size="+2"><b>{{eventData.name}}</b> will take place on <b>{{eventData.startDate}}</b> from <b>{{eventData.timeStart}}</b> to <b>{{eventData.timeEnd}}</b> and will be located in <b>{{eventData.location}}</b>.</font></p>
+    {% endif %}
+            <div>
+                <div>
+                    <font size="+2"><h3><br>Description:</h3>{{eventData.description}}<br><br>
+                    <div>
+                        {% if eventData.isService %}
+                            This event earns service hours.
+                        {% elif not eventData.isService %}
+                            This event does not earn service hours.
+                        {% endif %}
+                        {% if eventData.isRsvpRequired %}
+                            This event requires RSVP.
+                        {% elif not eventData.isRsvpRequired %}
+                            This event does not require RSVP.
+                        {% endif %}
+                        {% if eventData.recurringId %}
+                            This event is a recurring event.
+                        {% endif %}
+                        {% if eventData.isTraining %}
+                            This event is a training.
+                        {% endif %}
+                    </div>
+                </div>
+            </div>
+            <div class="container">
+              <div class="row">
+                <div class="col-sm">
+                    <h3><br>Facilitators:</h3>
+                        {% if eventFacilitatorNames %}
+                            <ul>
+                                {% for facilitator in eventFacilitatorNames%}
+                                    <li>{{facilitator.firstName}} {{facilitator.lastName}}</li>
+                                {% endfor %}
+                            </ul>
+                        {% else %}
+                            <div>No facilitators.</div>
+                        {% endif %}
+                    </div>
+                    <div class="col-sm">
+                        {% if programManager %}
+                            <div><h3><br>Program Manager:</h3></div>
+                                <div>
+                                    {{programManager.user.firstName}} {{programManager.user.lastName}}<br>
+                                    {{programManager.user.email}}
+                                </div>
+                        {% endif %}
+                    </div>
+                </div>
+                <div>
+                    <div class="row">
+                      <div class="col-sm">
+                          {% if program != None %}
```

Comment:
Just use `if program`. It's clearer than "if program not equal to None"

Reply:
"if program" doesn't work here so we are keeping this the same.